go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/kubeflow/manifests
```

Comment:
shouldn't we have the full go.mod?

Reply:
not sure. I ran GO111MODULE=on go mod tidy....and it seems this is the only dependency we need.....which is good.

Reply:
removed go get ...